src/main/java/com/myfirstownapp/beeapp/harvest/Harvest.kt

```diff
@@ -0,0 +1,17 @@
+package com.myfirstownapp.beeapp.harvest
+
+import javax.persistence.*
+
+@Entity
+@Table(name = "harvest")
+data class Harvest (
+        @Id
+        @GeneratedValue(strategy = GenerationType.AUTO)
+        val id: Long = 0,
+        @Column(name = "kind")
+        val kind: String = "",
```

Comment:
I think it`s better to use Kind as enum instead of String. Probably you'll need an adapter to parse it